keycloak/themes/hmda/login/template.ftl

```diff
@@ -60,6 +60,7 @@
               <div class="usa-alert-body">
                 <h3 class="usa-alert-heading">${message.type} Status</h3>
                 <p class="usa-alert-text">${message.summary}</p>
+                <p>For help account-related issues, please contact <a href="mailto:${properties.supportEmail!}">${properties.supportEmail}</a></em>.</p>
```

Comment:
I stand corrected. After asking around a bit it seems like most people prefer the `mailto:` over nothing at all.